Assets/Scripts/Entity/Player.cs

```diff
@@ -35,6 +38,27 @@ void OnTriggerEnter2D(Collider2D other)
         public void changeCounter() {
             counter.text = "Collectibles: " + collectible;
         }
+
+        void OnCollisionEnter2D(Collision2D collisionEnemy)
```

Comment:
Have this as a single method called for example `Hurt` that takes in vector 2d for enemy bump. If player health reaches 0, send back to spawn point

Reply:
Not quite sure what is needed here. More input?